### ISSUE TITLE
How fast when you training the COCO? 

### ISSUE THREAD

**Issue by ducheng678** [princeton-vl/CornerNet-Lite#39]
I get more than 200.0s/it. 
It`s too slow for training a detection model.

**Comment by wxy1234567** [comment]
It's really too slow. My GPU is 1080ti. It can be reached 1.4s/it.The author's GPU is 10 Titan X

**Comment by ducheng678** [comment]
Do you train the CornerNet_Saccade too?

**Comment by wxy1234567** [comment]
CornerNet

**Comment by ducheng678** [comment]
Could you try the CornerNet_Saccade?
My GPUs are 4 tesla p100, but I have no idea about why it`s so slow.
Thx a lot.

**Comment by wxy1234567** [comment]
I changed the batch_size because I didn't have enough memory. You can try to modify the parameters in CornerNet_Saccade.json.

**Comment by ducheng678** [comment]
I try training the CornerNet using one GPU, then I get about 1.4s/it.
It feels like that communication between GPUs takes more time.
And it takes more time to train CornerNet_Saccade than CornerNet.

**Comment by wxy1234567** [comment]
CornerNet_Saccade parameter is smaller than CornerNet parameter. The specific reason is waiting for the author to reply. I just did a simple test without looking at the code.

**Comment by ducheng678** [comment]
Yeah, I find that the augmentation parameter "ref_scale" is true when training CornerNet_Saccade but false in CornerNet. Let's wait for the official reply.

**Comment by heilaw** [comment]
It was around ~2s/it when I trained CornerNet-Saccade on 4 1080Ti GPUs. [Others]( https://github.com/princeton-vl/CornerNet-Lite/issues/19) have obtained around ~1.43/s it when training it on 2 1080Ti GPUs.

I am not quite sure what causes it trains so slow on your machine. Can you check what's the CPU memory usage or swap usage when you train CornerNet-Saccade?

**Comment by QQ2737499951** [comment]
I train CornerNet-Saccade，but stoped in 0%

**Comment by zhangli-0812** [comment]
I got 1.52s/it while training CornerNet-Saccade on 1 Titan X.

**Comment by BCWang93** [comment]
> I try training the CornerNet using one GPU, then I get about 1.4s/it.
> It feels like that communication between GPUs takes more time.
> And it takes more time to train CornerNet_Saccade than CornerNet.

Hi.Did you changed some parameters in using one gpu?Btw,do you know how to change the used gpu number?How can I change the parameters about batch_size?Thank you very much!

**Comment by BCWang93** [comment]
I got 2.26s/it when training CornerNet in one Tesla M40! And it looks like need 300 hours to train it!
